### ISSUE TITLE
Classes get partially replaced when they share some similarity in HTML (match exact wording)

### ISSUE THREAD

**Issue by la5digital** [n4j1Br4ch1D/postcss-obfuscator#9]
# PLZ ignore this and check comments starting from: https://github.com/n4j1Br4ch1D/postcss-obfuscator/issues/9#issuecomment-1542463798
----

See json output: https://www.diffchecker.com/5Q0M3X3U/

1) Original (left) with the following options (results in broken design and classes like md: and dark: not obfuscated properly):
```
            fresh: true,
            jsonsPath: 'css-obfuscator',
            srcPath: "resources", 
            desPath: "public",
            extensions: ['.html', '.php'],
            cssExcludes: [],
```

2) Json file generated correctly with the following options added (but no obfuscation aside from classSuffix):
```
            classMethod:  'none',
            classSuffix: "-" + getRandomName(6),
            formatJson: true, 
```

In conclusion, classes such as lg:, md:, dark: don't appear to be obfuscating (as it appears with difference in json output). The matching obfuscated class doesn't include the lg:, md:, dark: prefixes. 

Using latest version: v1.5.4.

Related to #6 ?

**Comment by n4j1Br4ch1D** [comment]
you referd to issue #6:
in JSON:
```
  ".dark\\:divide-gray-700": ".dark\\:divide-gray-700-mk1ljb",
```` 
having two extra backslashes is ok.
having them in CSS or HTML is not ok

**Comment by la5digital** [comment]
Hi @n4j1Br4ch1D 

Haven't spent too much time debugging but have noticed a few differences (mainly the changes in HTML not being correct).

1) Original class min-h-screen is mapped to '.ss9kq' in the json:
`  ".min-h-screen": ".ss9kq",`

However on the dist html file, it is renamed to "min-u6yxj" and partially mapped to the .u6yxj class which shows as following in json file:
`  ".h-screen": ".u6yxj",`

**Comment by n4j1Br4ch1D** [comment]
@la5digital This fix will be released in version `1.6.0` coming soon. give it a try then thanks for your patience.
